sketch-canvas-addon/src/main/java/org/vaadin/SketchCanvas.java

```diff
@@ -33,8 +33,8 @@
    * Image data consumer
    * @param <String> svg or base64 image
```

Comment:
mvn clean install does not work anymore for root project: it fails when it generates javadocs, one reason might be that @param tag is wrong <String> should be <T>..  Can check other javadocs too

Reply:
Otherwise looks good